### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -29,10 +19,46 @@ cxx_compiler:                  # [win]
   - vs2015                     # [win]
 CONDA_BUILD_SYSROOT:           # [osx]
   - /opt/MacOSX10.9.sdk        # [osx]
+macos_min_version:
+  - 10.9
+macos_machine:
+  - x86_64-apple-darwin13.4.0
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: @msarahan, are the 3 options above used? They are removed by conda-smithy

**Comment by isuruf** [comment]
@conda-forge-linter, please lint

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (msarahan) [reply recipe/conda_build_config.yaml]: they will be used when you start using the Anaconda compilers.  Currently, they are not.

Comment (isuruf) [reply recipe/conda_build_config.yaml]: Then I guess we have to mark them as used in conda-smithy right? Or will they be marked as used automatically if the recipe used new compilers?

Comment (msarahan) [reply recipe/conda_build_config.yaml]: they'll probably need to be added to some kind of "always used" list either in conda-smithy or in conda-build.  I don't know if they actually get detected, or if they're more indirect.

**Comment by jakirkham** [comment]
Would this be a good time to pull in the current pinnings from the webpage pinning script? We should also probably delete that script or add a big note that people should add pinnings over here going forward.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/conda_build_config.yaml]: @msarahan, what should `CONDA_BUILD_SYSROOT` be set to?

Comment (msarahan) [reply recipe/conda_build_config.yaml]: You have to install the appropriate OSX SDK.  The value of CONDA_BUILD_SYSROOT is where you have that extracted.

Comment (isuruf) [reply recipe/conda_build_config.yaml]: @jakirkham, do you know where it is installed in travis-ci?

Comment (msarahan) [reply recipe/conda_build_config.yaml]: Here's one place you can get them: https://github.com/phracker/MacOSX-SDKs

Comment (msarahan) [reply recipe/conda_build_config.yaml]: We will be updating to 10.10 sometime soon, I think - but we'll give you a heads-up when we're going to do it.

Comment (isuruf) [reply recipe/conda_build_config.yaml]: I guess we can install it into `/opt/MacOSX10.9.sdk` which is the value set here currently.

**Comment by jakirkham** [comment]
Merging contents over in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7 ). This is pretty close to the same. Could use more eyes though.

**Pull request merged**